src/x-components/plugin.options.ts

```diff
@@ -17,8 +17,7 @@ export const installXOptions: InstallXOptions = {
       messages
     });
     vue.use(i18n);
-    (window as any).setLocale = i18n.setLocale.bind(i18n); // Only for testing
```

Comment:
I'll leave the `setLocale` in the window as it could be used in some tests.

Reply:
but you can change it using window.initX.lang

Reply:
If you remove this binds, you caan't right?